shell/components/SortableTable/paging.js

```diff
@@ -7,11 +7,11 @@ export default {
     },
 
     indexTo() {
-      return Math.min(this.filteredRows.length, this.indexFrom + this.perPage - 1);
+      return Math.min((this.listLength || this.filteredRows.length), this.indexFrom + this.perPage - 1);
```

Comment:
Ideally, this shouldn't be in the SortableTable to begin with. It does look a little messy but isn't especially brittle, could be refactored as a computed property. Also probably worth adding some more logic to bypass more computed properties since most of the logic in the advanced worker makes these mixins redundant.